tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/SeqModelChecker.scala

```diff
@@ -106,6 +122,23 @@ class SeqModelChecker[ExecutorContextT](
     listeners.foreach(_.onCounterexample(rootModule, trace, invViolated, errorIndex))
   }
 
+  /**
+   * Notify all listeners that an example should be processed.
+   *
+   * @param rootModule
+   *   The checked TLA+ module.
+   * @param trace
+   *   The counterexample trace.
+   * @param index
+   *   Number of found error (likely [[SearchState.nRunsLeft]]).
+   */
+  private def notifyExample(
```

Comment:
Otherwise it sounds like the example itself is what is being notified.
```suggestion
  private def notifyOfExample(
```